charts/jellyfin/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v1
 appVersion: v10.5.3-ls45
 description: Jellyfin is a Free Software Media System
 name: jellyfin
-version: 1.0.0
+version: 1.0.1
```

Comment:
It looks like this file on the master branch shows `1.0.1`, so do you mind changing this to `1.0.2`?

Reply:
Sure :+1: